tools/cpplint.py

```diff
@@ -1040,7 +1040,7 @@ def ParseNolintSuppressions(filename, raw_line, linenum, error):
                 'Unknown NOLINT error category: %s' % category)
 
 
-def ProcessGlobalSuppresions(lines):
+def ProcessGlobalSuppressions(lines):
```

Comment:
This should be upstreamed https://github.com/cpplint/cpplint/blob/b29d3ee08e5ba1585506f220e588e2d2907f3bf7/cpplint.py#L1114

Reply:
In the upstream, it looks like they redirect the typo to the correct function

Reply:
I'd be ok approving the PR if the update to tools/cpplint.py was removed. I think that one should be in a separate PR which would be to sync the file with the current ccplint upstream.

Reply:
Sure thing, FWIW those typos are already fixed upstream and were released just two days ago in cpplint v2.0.0.